frontend_tests/zjsunit/namespace.js

```diff
@@ -21,7 +22,7 @@ exports.patch_builtin = function (name, val) {
 
 exports.zrequire = function (name, fn) {
     if (fn === undefined) {
-        fn = 'js/' + name;
+        fn = path.resolve(__dirname, "../../static/js", name);
```

Comment:
I guess this `path.resolve` shouldn’t be needed.

Reply:
Yep, this isn't require because we set the `NODE_PATH` environment variable in [tools/test-js-with-node](https://github.com/zulip/zulip/blob/master/tools/test-js-with-node#L110) to take care of this.

Reply:
@priyank-p That’s not why (I’m removing `NODE_PATH` from `test-js-with-node` as part of this commit). It’s because the `require()` below this line is already relative to `__dirname`.

Reply:
Oh, ok! I was just going by the comment!